### PR TITLE
Competing rank indicator is filtered to only available positions

### PR DESCRIPTION
**Pull request by mjoyce91**
- The "has competing rank" indicator in the short list is now filtered to only available positions that are in an active bid cycle (use mock PR below)
- Unrelated, but also updates BidHandshake model to include the bid cycle ID, as we may want to do similar lookups to check if handshakes are in an active bid cycle without sending the universe of cp_ids that we store

Use with https://github.com/MetaPhase-Consulting/mock-fsbid/pull/206

### CODE REVIEW

File: talentmap_api/fsbid/services/reference.py [hunk 1]
```diff
@@ -4,11 +4,30 @@
...
+from talentmap_api.fsbid.views import reference as views
+
 API_ROOT = settings.FSBID_API_URL
 
 logger = logging.getLogger(__name__)
 
 
+def get_cycles(jwt_token):
+    '''
+    Gets an indivdual available position by id
```
Comment (elizabeth-jimenez) [inline talentmap_api/fsbid/services/reference.py:17]: Wrong description?

**Review by elizabeth-jimenez — CHANGES_REQUESTED** [review]
Just a comment/question about the `get_cycles` description

**Comment by mjoyce91** [comment]
@elizabeth-jimenez - updated

**Review by elizabeth-jimenez — APPROVED** [review]
🍰 👍

**Pull request merged**